packages/govuk-frontend/src/govuk/components/back-link/template.jsdom.test.js

```diff
@@ -35,32 +35,74 @@ describe('back-link component', () => {
     expect($component).toHaveClass('app-back-link--custom-class')
   })
 
-  it('allows the link to be customised using the `href` option', () => {
-    document.body.innerHTML = render('back-link', examples['with custom link'])
+  describe('the `href` option', () => {
+    it('allows the link to be customised', () => {
+      document.body.innerHTML = render(
+        'back-link',
+        examples['with custom link']
+      )
+
+      const $component = document.querySelector('.govuk-back-link')
+      expect($component).toHaveAttribute('href', '/home')
+    })
 
-    const $component = document.querySelector('.govuk-back-link')
-    expect($component).toHaveAttribute('href', '/home')
+    it.each(['', 0, false, null, undefined])(
+      'displays `Back` for `%s`',
```

Comment:
I think this test is misnamed, as it appears to be testing whether `href` is `#` when a value is falsy or undefined, not the link text.

The test for the link text is on line 78.

Reply:
Whoops, good catch, I'll rename.